src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,74 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String KOMA = ",";
+    private static final String[] resaultArr = new String[]{SUPPLY, BUY, RESULT};
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] dataInArray = readData(fromFileName);
+        int[] finalValues = reportValues(dataInArray);
```

Comment:
```suggestion
        String report = createReport(dataInArray);
```
create a final report here

Reply:
not fixed